app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_incorrect_year_format() -> None:
+    with pytest.raises(ValueError):
+        get_human_age(-12, 43.1)
```

Comment:
The test case `test_incorrect_year_format` is intended to check for invalid input types. However, the current implementation might not trigger the `ValueError` for non-integer inputs as expected. Ensure that the `get_human_age` function properly handles non-integer inputs and raises an error.